Library/Homebrew/cmd/info.rb

```diff
@@ -162,12 +162,6 @@ def print_info
             info_formula(obj)
           when Cask::Cask
             info_cask(obj)
-          when FormulaUnreadableError, FormulaClassUnavailableError,
-             TapFormulaUnreadableError, TapFormulaClassUnavailableError,
-             Cask::CaskUnreadableError
```

Comment:
These are not possible members of the collection returned by `to_formulae_and_casks_and_unavailable`